src/lib/y2users/configuration.rb

```diff
@@ -21,20 +21,20 @@ module Y2Users
   # Holds references to elements of user configuration like users, groups or passwords.
   # Class itself holds references to different configuration instances.
   # TODO: write example
-  class Configuration
+  class Config
```

Comment:
Please, rename the file too. And also adapt all the files *requiring* this class.